arm-notificationhubs/2017-04-01/swagger/notificationhubs.json

```diff
@@ -0,0 +1,1956 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NotificationHubsManagementClient",
+    "description": "Azure NotificationHub client",
+    "version": "2017-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
```

Comment:
Could you please check and confirm if your service really supports text/json????